### DISCUSSION TITLE
App Store

### DISCUSSION THREAD

**Discussion by pqhf5kd** [msimms/OpenWorkoutTracker#45] [General]
Will we see an AppStore release of this?

**Comment by msimms** [comment]
I hope so. I'm currently rewriting the user interface in SwiftUI, which is going fairly quickly so....

**Comment by pqhf5kd** [comment]
That's great, I'm looking forward to trying it out.
It looks like it's not possible to build and run the project on device without a paid developer account.

**Comment by pqhf5kd** [comment]
Any progress on this @msimms?

**Comment by msimms** [comment]
I should just go ahead and get something out. I guess the quickest path to a minimum viable product would be to fix a few known bugs, do some more testing, and then submit it to the Apple Store.

I've been using the iOS app as a bike computer a lot so I know that part works just fine. I also regularly use the watch app when running. There's a few usability bugs and exporting to a .fit file is broken (.tcx and .tpx exports work though).

**Comment by msimms** [comment]
I should also look at building it on a machine where XCode that isn't tied to my developer account and make sure that's all good as well.

**Comment by msimms** [comment]
...and documentation

**Comment by pqhf5kd** [comment]
Any progress on this?
